h2o-docs/src/product/data-science/glm.rst

```diff
@@ -103,7 +103,7 @@ Defining a GLM Model
 
 -  `lambda <algo-params/lambda.html>`__: Specify the regularization strength.
 
--  `lambda_search <algo-params/lambda_search.html>`__: Specify whether to enable lambda search, starting with lambda max. The given lambda is then interpreted as lambda min. 
+-  `lambda_search <algo-params/lambda_search.html>`__: Specify whether to enable lambda search, starting with lambda max. If you also specify a value for ``lambda``, then this value is interpreted as lambda min. If you do not specify a value for ``lambda``, then GLM will calculate the minimum lambda. 
 
```

Comment:
not true. lambda min is set by lambda min ratio. provided lambda(s) are the only lambdas considered by the search.